hub/core/tensor.py

```diff
@@ -301,10 +308,7 @@ def info(self, value):
             raise TypeError("Info must be set with type Dict")
 
     @invalid_view_op
-    def append(
-        self,
-        sample: InputSample,
-    ):
+    def append(self, sample: InputSample, progressbar=False):
```

Comment:
Append only adds a single sample so I don't think we need a progress bar for it

Reply:
Got it. Will update!